wordpress.org/public_html/wp-content/plugins/plugin-directory/jobs/class-api-update-updater.php

```
@@ -567,6 +568,18 @@ public static function force_release( $plugin_slug, $reason, $user = null ) {
 			)
 		);
 
+		// Track what the force-release skipped: a readable reason total, and a per-plugin breakdown (groups display values, not a sum).
+		if ( function_exists( 'bump_stats_extra' ) && 'production' === wp_get_environment_type() ) {
+			if ( $was_blocked ) {
+				bump_stats_extra( 'plugin-force-release', 'block' );
+				bump_stats_extra( 'plugin-force-release-block', $post->post_name );
```

PHPStan (2.2.8)

[error] Cannot access property $post_name on WP_Post|true.


(property.nonObject)